outputs.tf

```diff
@@ -4,17 +4,17 @@
 
 output "vpc_name" {
   description = "Name of VPC created"
-  value       = ibm_is_vpc.vpc.name
+  value       = var.create_vpc == true ? var.name : data.ibm_is_vpc.vpc[0].name
 }
 
 output "vpc_id" {
   description = "ID of VPC created"
-  value       = ibm_is_vpc.vpc.id
+  value       = local.vpc_id
 }
 
 output "vpc_crn" {
   description = "CRN of VPC created"
-  value       = ibm_is_vpc.vpc.crn
+  value       = var.create_vpc == true ? resource.ibm_is_vpc.vpc[0].crn : data.ibm_is_vpc.vpc[0].crn
```

Comment:
Does this `resource.ibm_is_vpc.vpc[0].crn` work?? I would usually expect just `ibm_is_vpc.vpc[0].crn`